styles.css

```diff
@@ -0,0 +1,183 @@
+.wrapper{
+  margin: auto;
+  width: 80%;
+}
+.header{
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 100px;
+}
+.logo{
+  position: relative;
+  padding: 40px 0;
+  padding-left: 44px;
+  font: bold 24px/31px 'PT Serif',serif;
+}
+.logo__circle{
+  position:absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 28px;
+  height: 28px;
+  border-radius: 50%;
+  opacity: .7;
+}
+.logo__circle_color_blue{
+  left: 8px;
+  top: 56%;
+  background: #C8DBFB;
+}
+.logo__cirle_color_dark-blue{
+  background: #0056EC;
+}
+.logo, .menu, .burger{
+  display:inline-block;
+}
+.burger{
+  visibility:hidden;
+  width: 30px;
+  padding-top: 36px;
+}
+.burger__menu{
+  width: auto;
+  height: auto;
+  cursor: pointer;
+  background-color: #fff;
+  border: none;
+  transition: opacity .5s;
+}
+.burger__menu:hover{
+  opacity: .5;
+}
+.menu{
+  display: flex;
+  align-items: center;
+}
+.menu__item{
+  position: relative;
+  margin-right: 26px;
+  font: 500 18px/21px Roboto,arial,helvetica,sans-serif;
+  color: #000;
+  text-decoration: none;
+  transition: opacity .5s;
+}
+.menu__item:last-child{
+  margin-right: 0;
+}
+.menu__item:after{
+position: absolute;
+left: 0;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/site99/pullRequest?prid=6205931)